src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -271,7 +271,7 @@ def get_title_link(
     event: Event | None,
     link_to_event: bool,
     issue_details: bool,
-    notification: BaseNotification | None,
+    notification: OrganizationNotification | None,
```

Comment:
@mgaeta Not seeing any dependency on the organization here so I'm not sure why we need this change